readme.md

```diff
@@ -167,6 +173,12 @@ https://videopress.com/embed/*
 <iframe src="https://videopress.com/embed/zcnJVzQF" width="600" height="600"></iframe>
 ```
 
+**Google redirection url**
+
+```
+https://google.*/url?source=web&cd=1&ved=0ahUKEwj30L2MvpDVAhUFZVAKHb8CBaYQuAIIIjAA&url=*&usg=AFQjCNECyDn3DQL7U6VW2CnXQQjB0gNKqA
```

Comment:
It is misleading to use the * in the domain, because I have told the readers that the `*` means something specific,

so please change: `https://google.*` to `https://google.<extension>` or just any valid example like `https://google.cz`.

Also your example url uses the `url` parameter twice and it's really long, so I think we can condense this down to match the format of the other examples better:

```
https://google.<extension>/?url=https%3A%2F%2Fwww.youtube.com%2Fwatch%3Fv%3DeG1uDU0rSLw
```